solar/solar/utils.py

```diff
@@ -69,9 +69,20 @@ def generate_uuid():
     return str(uuid4())
 
 
-def render_template(template_path, params):
+def to_json(data):
+    return json.dumps(data)
+
+
+def to_pretty_json(data):
+    return json.dumps(data, indent=4)
+
+
+def render_template(template_path, **params):
+    env = Environment()
```

Comment:
Probably we should make it global, and not create full jinja2 env for every call.